ipi/engine/motion/neb.py

```diff
@@ -338,32 +321,46 @@ class NEBMover(Motion):
         qlist_lbfgs: list of previous positions (x_n+1 - x_n) for L-BFGS
         glist_lbfgs: list of previous gradients (g_n+1 - g_n) for L-BFGS
         endpoints: flag for minimizing end images in NEB *** NOT YET IMPLEMENTED ***
+        use_climb: flag for climbing image NEB
```

Comment:
please add a safe exit for features which are not implemented.

Reply:
Added a catch for variable springs. Other cases seem to be safe.